Tests/CatalogItems/Get-RsFolderContentTests.ps1

```diff
@@ -18,6 +18,8 @@ Describe "Get-RsFolderContent" {
         # Removing folders used for testing
         Remove-RsCatalogItem -ReportServerUri 'http://localhost/reportserver' -RsFolder $folderPath -Confirm:$false
     }
+"$($folderName)";
```

Comment:
Would you mind to remove this leftover?

Reply:
Will do.  Now that it{s passing, I think we can just remove it.